third_party/terraform/tests/resource_compute_region_disk_test.go

```diff
@@ -23,26 +23,38 @@ func TestAccComputeRegionDisk_basic(t *testing.T) {
 		Providers:    testAccProviders,
 		CheckDestroy: testAccCheckComputeRegionDiskDestroy,
 		Steps: []resource.TestStep{
-			resource.TestStep{
-				Config: testAccComputeRegionDisk_basic(diskName, "self_link"),
+			{
+				Config: testAccComputeRegionDisk_basic(diskName, "${data.google_self_link.snapdisk.self_link}"),
```

Comment:
Does `testAccComputeRegionDisk_basic` still need the second parameter, or could we get rid of it? I don't know if it's used elsewhere that isn't being replaced by an interpolation.